src/schnetpack/configs/experiment/mptrj.yaml

```diff
@@ -0,0 +1,58 @@
+# @package _global_
+
+defaults:
+  - override /model: nnp
+  - override /data: mptraj
+
+run:
+  id: mptraj_run
+  data_dir: /tmp
+  experiment: mptraj
+
+globals:
+  cutoff: 6.0
+  lr: 5e-4
+  property: energy
+  aggregation: sum
+
+data:
+  transforms:
+    - _target_: schnetpack.transform.SubtractCenterOfMass
+    - _target_: schnetpack.transform.RemoveOffsets
+      property: ${globals.property}
+      remove_atomrefs: True
+      remove_mean: True
+    - _target_: schnetpack.transform.MatScipyNeighborList
```

Comment:
I suggest to use `CachedNeighborList` as a wrapper for `MatScipyNeighborList` to make training faster. you can find [here](https://github.com/atomistic-machine-learning/schnetpack/blob/master/src/schnetpack/transform/neighborlist.py)